app/cafe.py

```diff
@@ -0,0 +1,23 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+
+    def __init__(self, name: str):
+        self.name = name
+
+    def visit_cafe(self, visitor: dict):
+        if 'vaccine' not in visitor:

```

Comment:
It's better to use the same quote style throughout the project
I suggest using double-quotes. Please, correct it :)